docs/source/guides/contributing.rst

```diff
@@ -7,12 +7,22 @@ ModdingDocs uses `reStructuredText <https://en.wikipedia.org/wiki/ReStructuredTe
 
 A cheatsheet for reStructuredText syntax can be found here: https://docs.generic-mapping-tools.org/6.2/rst-cheatsheet.html
 
+Contributing without a local build
+----------------------------------
+You don't necessarily need to set up a local build environment.
+
+To contribute without doing so, you cant just edit the files in an editor of your choice and create a github PR from them.
```

Comment:
```suggestion
To contribute without doing so, you can just edit the files in an editor of your choice and create a GitHub pull request from them.
```

Typo and spelling out "pull request" for those potentially unfamiliar with the abbreviation "PR"